sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -28,8 +29,8 @@ import sbtprotoc.ProtocPlugin.autoImport.PB
 import scalapb.GeneratorOption
 
 object AkkaserverlessPlugin extends AutoPlugin {
-  override def trigger = allRequirements
-  override def requires = ProtocPlugin
+  override def trigger = noTrigger
+  override def requires = ProtocPlugin && AkkaGrpcPlugin
```

Comment:
could you also add the `akkaserverless-scala-sdk` to libraryDependencies while you are at it? (we have a FIXME in the samples for that)

Reply:
I thought that should already happen automatically because of https://github.com/lightbend/akkaserverless-java-sdk/blob/main/codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/AkkaserverlessGenerator.scala#L58 - will verify

Reply:
> will verify

Yes, seems to work, added #465